packages/client/components/ReflectionCard/ReflectionCard.tsx

```diff
@@ -254,11 +255,25 @@ const ReflectionCard = (props: Props) => {
     !isComplete &&
     !isDemoRoute() &&
     (isHovering || !isDesktop)
+
+  const reflectionCount = reflectionGroups.reduce(
+    (sum, {reflections}) => sum + reflections.length,
+    0
+  )
+
+  const shouldAnimate = () => {
+    if (phaseType !== 'group') return false
+    if (reflectionCount !== reflectionGroups.length) return false
+    if (reflectionId !== reflectionGroups[0]?.reflections[0]?.id) return false
```

Comment:
-1 If you want to be sure that the value of `reflectionGroups[0]` exists, it might also need another layer of [Optional chaining (?.)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)?
```js
reflectionGroups?.[0]?.
```

But it's still a good idea to check the TypeScript declaration of `reflectionGroups` and `reflections` directly here.